package-lock.json

```diff
@@ -104,10 +104,22 @@
       "version": "github:brave/ad-block#b933cf5168c5284dc159735d8bcefb415d0cfc00",
       "from": "github:brave/ad-block#b933cf5168c5284dc159735d8bcefb415d0cfc00",
       "requires": {
-        "bloom-filter-cpp": "^1.2.2",
+        "bloom-filter-cpp": "^1.2.0",
```

Comment:
It would be best to not version this `package-lock.json` file at all for this PR